src/zeroD/IdealGasMoleReactor.cpp

```diff
@@ -0,0 +1,237 @@
+//! @file IdealGasMoleReactor.cpp A constant volume zero-dimensional
+//! reactor with moles as the state
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#include "cantera/zeroD/IdealGasMoleReactor.h"
+#include "cantera/zeroD/FlowDevice.h"
+#include "cantera/zeroD/ReactorNet.h"
+#include "cantera/zeroD/ReactorSurface.h"
+#include "cantera/kinetics/Kinetics.h"
+#include "cantera/thermo/ThermoPhase.h"
+#include "cantera/thermo/SurfPhase.h"
+#include "cantera/base/utilities.h"
+#include "float.h"
```

Comment:
I have found the `float.h` include to be necessary because I am calling the `__DBL_EPSILON__` from that header which is more consistent across platforms.